### PR TITLE
ADEV 33 arrow back to top

### PR DESCRIPTION
**Pull request by LarryMatte**

### CODE REVIEW

File: site/src/main/site/assets/js/global.js [hunk 1]
```diff
@@ -10,6 +10,19 @@ $(function () {
             $(this).addClass("closed");
         });
 
+    // Arrow back to top in the footer
+    $(window).scroll(function(){
+        if ($(this).scrollTop() > 100) {
```
Comment (spg) [inline site/src/main/site/assets/js/global.js]: why exactly `100` ?

**Comment by meriouma** [comment]
LGTM

File: site/src/main/site/assets/js/global.js (hunk 1 above)
Comment (christiangoudreau) [inline site/src/main/site/assets/js/global.js]: Magic number anyway, extract as a variable

Comment (LarryMatte) [inline site/src/main/site/assets/js/global.js]: @spg I changed it to 200, that way the arrow will appear right after the first line of text is scrolled out of the window

Comment (spg) [inline site/src/main/site/assets/js/global.js]: @LarryMatte The idea is to refactor the `200` into a variable, so we can know better what `200` refers to, e.g.: `int usefulVariableName = 200;`

Comment (LarryMatte) [inline site/src/main/site/assets/js/global.js]: oOh I get it

**Comment by spg** [comment]
LGTM

**Comment by Pacane** [comment]
LGTM

**Pull request merged**